test/controllers/email/VerifyPasscodeControllerSpec.scala

```diff
@@ -284,10 +296,13 @@ class VerifyPasscodeControllerSpec extends ControllerBaseSpec with MockEmailVeri
 
       "there is an email in session" should {
 
-        lazy val result = TestVerifyPasscodeController.updateEmailAddress()(requestWithEmail)
+        lazy val result = {
```

Comment:
Can you also add tests for the `updateEmailAddress` action for the different scenarios when calling the service function to update the email. Current scenarios untested are: success response but empty message, conflict response, unexpected error response.